src/components/TodoList.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import cn from 'classnames';
+import { TodoItem } from './TodoItem';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList:React.FC<Props> = ({ todos }) => (
+  <section
+    className={cn('todoapp__main', {
+      hidden: todos.length === 0,
+    })}
+  >
+    {todos.length > 0 && todos.map(todo => (
+      <TodoItem todo={todo} />
```

Comment:
don't forget about `key`

Reply:
I noticed it lately and added key in next tasks
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/65409847/ef5432ca-8d54-4c07-bc5b-6b4ab01adf32)